exec/java-exec/src/main/java/org/apache/drill/exec/server/options/InMemoryOptionManager.java

```diff
@@ -35,6 +37,20 @@
   }
 
   @Override
+  public void deleteOption(final String name, final OptionValue.OptionType type) {
+    throw UserException.unsupportedError()
```

Comment:
Why throw a UserException here ? is the error message relevant to the user ? do we even expect this method to be called ?